src/web/api/src/main/java/org/locationtech/geogig/rest/repository/InitRequestUtil.java

```diff
@@ -256,8 +255,7 @@ private void updateHintsWithParams(Hints hints, Map<String, String> params)
         }
     }
 
-    @VisibleForTesting
-    Hints createHintsFromRequest(Request request) throws UnsupportedEncodingException, URISyntaxException, IOException,
+    public Hints createHintsFromRequest(Request request) throws UnsupportedEncodingException, URISyntaxException, IOException,
```

Comment:
The static INSTANCE variable of this class is not used.  Perhaps this should simply be a static function and called via `InitRequestUtil.createHintsFromRequest(...)`.